tests/test_profile_parsing.py

```diff
@@ -50,6 +55,11 @@ def test_loading_from_influx(self):
         )
         problem.pre()
 
+        # The timezone setting is dependand on what is specificied
+        np.testing.assert_equal(
+            problem.io.reference_datetime.tzinfo, pytz.timezone("UTC") or datetime.timezone.utc
```

Comment:
I don't get this, why do we need the "or" statement here, why is it needed? Can we get rid of the pytz here?

Reply:
@jimrojerTNO it has been removed